arangod/Cluster/AgencyPaths.h

```diff
@@ -1603,7 +1603,7 @@ class Root : public std::enable_shared_from_this<Root>, public Path {
 
           using BaseType::DynamicComponent;
 
-          class TransactionId : public StaticComponent<TransactionId, Server> {
```

Comment:
these changes don't look intentional but rather like from a global search&replace.
the same comment applies for the following replacements of `TransactionId` with `TRI_voc_tid_t`.

Reply:
Oh, shit, yeah that's my bad. Was messing around with a different checkout and must have accidentally misfired.